governance/working-group-chair-proposal-process.md

```diff
@@ -0,0 +1,82 @@
+# Environmental Sustainability TAG Working Group Chair Proposal Process
+
+TAG Environmental Sustainability Working Group Chairs operate as a team alongside the TAG's co-chairs and technical leads. 
+They provide guidance and directions to the working group members and ensure that the working group's efforts are in line with it's mission and values, as well as the overall goals of the TAG. They facilitate the working group's activities and coordinate its members' efforts. They schedule working group meetings, set meeting agenda, and maintain effective communication to ensure smooth collaboration among the working group's participants.
+Finally, they provide regular updates to the TAG, keeping the TAG leadership team and members informed about the group's progress and deliverables.
+
+## Caveats of Working Group Chairs
+
+* The position of a working group chair is not a paid position (TAG co-chairs and tech leads are also not paid positions)
+* Working group Chairs [have responsibilities](roles.md) to the Environmental Sustainability TAG, the leadership team, and the TOC
+* Nominations by the community, with endorsements by 1 or more working group/TAG co-chairs
+
+## Process of nominations
+
+Depending on the needs of the community, as determined by the Environmental Sustainability TAG leadership team, working group chair nominations may come in the form of "Nominees by community", "Nominees by working group co-chairs", "Nominees by TAG co-chairs" or all of them.
+
+The Environmental Sustainability TAG does not have a current limit on the amount of working group chairs.
+
+Final working group chair nominations will adhere to the [CNCF's TOC described process](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md#elections).
+
+Nominees should adhere to the working group chair nomination requirements described below.
+Any individual who is a member of the Environmental Sustainability TAG list or the tag-environmental-sustainability slack channel may nominate another individual.
+
+The working group co-chairs reserve the right to endorse a nominee by the community based on:
+
+* Gender diversity
+* Company diversity
+* Geo diversity
+* Amount of nominations from different companies, people, regions for a single individual
+* Existing body of work
+* Justification provided during nomination
```

Comment:
Recommend adjusting this slightly. The current text states that endorsements may come from the WG chairs or TAG chairs (recommend also including tech leads). I would reword this to the effect that endorsement and selection of individuals will consider the following to ensure appropriate community representation and technical acumen.

Reply:
Thanks, updated this section!